lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -176,6 +176,18 @@ def cmd_color(*args)
     driver.update_prompt
   end
 
+  #
+  # Tab completion for the color command
+  #
+  # @param str [String] the string currently being typed before tab was hit
+  # @param words [Array<String>] the previously completed words on the command line.  words is always
+  # at least 1 when tab completion has reached this stage since the command itself has been completed
+
+  def cmd_color_tabs(str, words)
+    return [] if words.length > 1
+    ['auto','true','false']
```

Comment:
Consider using `%w[]` for your string arrays. A bit more readable for tab completion code.